account/records.go

```diff
@@ -26,21 +27,29 @@ func CopyRecords(records Records) Records {
 
 // Record represents a single account record.
 type Record struct {
-	ID             uint32
-	Name           string
-	Email          string
+	ID              uint32
+	Name            string
+	Email           string
+	ClientViewHosts []string
+	DateCreated     string
+
+	// ClientViewURLS is used only by Version 2 clients. It is DEPRECATED
+	// and will go away when Version 2 is not longer supported.
```

Comment:
`not longer` -> `no longer`

Reply:
done in https://github.com/rocicorp/diff-server/pull/90